src/java.base/share/classes/java/nio/MappedMemoryUtils.java

```diff
@@ -94,7 +96,12 @@ static void force(FileDescriptor fd, long address, boolean isSync, long index, l
         } else {
             // force writeback via file descriptor
             long offset = mappingOffset(address, index);
-            force0(fd, mappingAddress(address, offset, index), mappingLength(offset, length));
+            try {
+                force0(fd, mappingAddress(address, offset, index),
```

Comment:
Are you going to change force0 to declare that it throws IOException? I assumed you'd need to do that to get this code to compile.

Reply:
It compiled without that change. In general I try to at least compile if not completely test before creating a PR. It should be changed however.